src/js/routes/factories/network.js

```diff
@@ -111,7 +111,8 @@ let RouteFactory = {
                 parentCrumb: 'virtual-networks-tab-detail-tasks-detail',
                 getCrumbs: function () { return []; }
               }
-            }
+            },
+            title: 'Details'
```

Comment:
Problem with that is, that we end with various place, which one has to update only to add a new tab. Thats very error prone.